src/main/java/frc/robot/Constants.java

```diff
@@ -24,6 +24,17 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+  public static final int TEAM_NUMBER = 8248;
```

Comment:
Nit: these are used for network tables only right? Ideally we'd put these in their own inner class specific to network tables.